### PR TITLE
Issue #318 [Multi-Language][Kotlin][Scala] Add directories to classpath to allow cross language compilation

### PR DESCRIPTION
**Pull request by cmsr**
Adding feature request by Issue #318 

Signed-off-by: Carlos Rodrigues <cmsrodrigues@gmail.com>

### CODE REVIEW

**Comment by davidB** [comment]
Thanks for the contribution.

**Comment by cmsr** [comment]
@davidB you are welcome! Do you think it would be considered for integration in a next version?

**Comment by davidB** [comment]
yes, I could release few days after the merge

File: src/main/java/scala_maven/ScalaCompileMojo.java [hunk 1]
```diff
@@ -37,6 +39,14 @@
     @Parameter(property = "analysisCacheFile", defaultValue = "${project.build.directory}/analysis/compile")
     protected File analysisCacheFile;
 
+    /**
+     * Comma seperated list of directories or jars to add to the classpath
+     *
+     */
+    @Parameter(property="addToClasspath")
+    protected String addToClasspath;
+
```
[the remark this answers is not in the document]
Comment (cmsr) [reply src/main/java/scala_maven/ScalaCompileMojo.java]: I do agree with a more typed implementation. I made a refactoring considering that.

**Pull request merged**